exporter/instanaexporter/README.md

```diff
@@ -15,7 +15,7 @@ The following exporter configuration parameters are supported.
 
 | Parameter      | Description |
 |----------------|-------------|
-| endpoint | The Instana backend endpoint that the Exporter connects to. It depends on your region and it starts with ``https://serverless-``. It corresponds to the Instana environment variable ``INSTANA_ENDPOINT_URL`` |
+| endpoint | The Instana backend endpoint that the Exporter connects to. It depends on your region and it starts with ``https://serverless-`` for SaaS. For on prem, make sure to use ``https://``. It corresponds to the Instana environment variable ``INSTANA_ENDPOINT_URL`` |
```

Comment:
It depends on your region and it starts with ``https://serverless-`` for SaaS. For on prem, make sure to use ``https://``. --> It depends on your region and how it is hosted. It starts with ``https://serverless-`` for SaaS. Otherwise, it starts with ``https://``.

Reply:
I am suggestion the above because it could be `https://` even if not on prem.